### ISSUE TITLE
unhelpful output when only small files

### ISSUE THREAD

**Issue by unhammer** [markfasheh/duperemove#187]
I found it really confusing why duperemove would sometimes just show me the Usage instead of trying to duperemove, I was sure I was messing up file name quoting or something, but even simple file names made it happen:
```
$ echo hi> one; echo hi>two

$ duperemove one two
duperemove v0.10
Find duplicate extents and print them to stdout

Usage: duperemove [-r] [-d] [-h] [--debug] [--hashfile=hashfile] OBJECTS

"OBJECTS" is a list of files (or directories) which we
want to find duplicate extents in. If a directory is 
specified, all regular files inside of it will be scanned.

        <switches>
        -r              Enable recursive dir traversal.
        -d              De-dupe the results - only works on btrfs.
        -h              Print numbers in human-readable format.
        --hashfile=FILE Use a file instead of memory for storing hashes.
        --help          Prints this help text.

Please see the duperemove(8) manpage for more options.
```

and then noticed --debug:
```
$ duperemove --debug one two
Skipping small file one
Skipping small file two
duperemove v0.10
Find duplicate extents and print them to stdout

Usage: duperemove [-r] [-d] [-h] [--debug] [--hashfile=hashfile] OBJECTS

"OBJECTS" is a list of files (or directories) which we
want to find duplicate extents in. If a directory is 
specified, all regular files inside of it will be scanned.

        <switches>
        -r              Enable recursive dir traversal.
        -d              De-dupe the results - only works on btrfs.
        -h              Print numbers in human-readable format.
        --hashfile=FILE Use a file instead of memory for storing hashes.
        --help          Prints this help text.

Please see the duperemove(8) manpage for more options.

```

– wouldn't it make more sense to say "only small files given, exiting" instead?

**Comment by Hi-Angel** [comment]
I think it's fixed, the git version it doesn't show man page anymore:

	$ echo hi> one; echo hi>two
	$ duperemove one two
	Using 128K blocks
	Using hash: murmur3
	Gathering file list...
	No dedupe candidates found.

> wouldn't it make more sense to say "only small files given, exiting" instead?

I'd say it rather makes sense to actually deduplicate the files, but [I have opened a distinct bug on this](https://github.com/markfasheh/duperemove/issues/196).

**Comment by markfasheh** [comment]
Closing as this particular behavior is fixed in duperemove v0.11
